docs/how-to/ccv0.sh

```diff
@@ -430,50 +419,29 @@ kubernetes_delete_ssh_demo_pod() {
 }
 
 crictl_sandbox_name=kata-cc-busybox-sandbox
-crictl_create_cc_pod() {
+unshadow_crictl_create_cc_pod() {
     # Update iptables to allow forwarding to the cni0 bridge avoiding issues caused by the docker0 bridge
     sudo iptables -P FORWARD ACCEPT
     
     # Create crictl pod config
-cat << EOF > ${HOME}/pod-config.yaml
-metadata:
-  name: ${crictl_sandbox_name}
-EOF
+    local pod_config="${FIXTURES_DIR}/pod-config.yaml"
 
-    # If already exists then delete and re-create
-    if [ -n "$(sudo crictl pods --name ${crictl_sandbox_name} -q)" ]; then
-        crictl_delete_cc
-    fi
-
-    local pod_id=$(sudo crictl runp -r kata ${HOME}/pod-config.yaml)
-    sudo crictl pods ${pod_id}
+    crictl_delete_cc_pod_if_exists "${crictl_sandbox_name}"
+    crictl_create_cc_pod "${pod_config}"
+    sudo crictl pods
 }
 
-crictl_create_cc_container() {
+unshadow_crictl_create_cc_container() {
```

Comment:
The `unshadow` prefix is confusing. I think that `unshadow` here refers to the same name in `lib.sh`, but you are not really unshadowing it, more like wrapping it to make it easier to use. It's also a bit inconsistent with the `crictl_delete_cc` case. What about a `wrap` or `call` prefix?